src/plone/restapi/services/components/get.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from plone.restapi.services import Service
+from zope.component import getMultiAdapter
+from zope.interface import implements
+from zope.publisher.interfaces import IPublishTraverse
+
+
+class ComponentsGet(Service):
+
+    implements(IPublishTraverse)
+
+    def __init__(self, context, request):
+        super(ComponentsGet, self).__init__(context, request)
+        self.params = []
+
+    def publishTraverse(self, request, name):
+        # Consume any path segments after /components_ as parameters
+        for component_id in name.split(','):
+            self.params.append(component_id)
+        return self
+
+    @property
+    def _component_ids(self):
+        return self.params
+
+    def _frame_component_items(self, items, component_id):
```

Comment:
I hurriedly picked the name for this method during the sprint, where the aim was to just cobble together a quick mock. Now I think it might cause confusion with the concept of JSON-LD framing. Maybe `_wrap_component_items()`?